### PR TITLE
added readme for slates

### PR DESCRIPTION
**Pull request by jarumihooi**
fixes partially #23 
added slates readme

### CODE REVIEW

File: january-slates/readme.md [hunk 1]
```diff
@@ -1 +1,106 @@
...
+    Project Version: Unknown/1.0  
+    Number of Batches: 1
+    Other Version Control Information: None
+```
+
+### Data Overview
+INPUT - a set of video files to be annotated. _TODO: Which tool?_   
+INTERMEDIATE RAW OUTPUT - `.csv` file where each line is the time of when the slate frames appear in that video. 
+```
+FORMAT:
```
Comment (keighrim) [inline january-slates/readme.md:38]: "FORMAT" being outside of code block?

File: january-slates/readme.md [hunk 2]
```diff
@@ -1 +1,106 @@
...
+GUID,",","Series/Group              ,","Slate Start ,","Slate End   ,","Writing Types,",Recorded/Digital,,,",",format of most of the information,",",Anything moving on screen during slate?,
+Note the extra commas added in and extra blank columns and extra space within the column names. 
+```
+```
+EXAMPLE:
```
Comment (keighrim) [inline january-slates/readme.md:43]: Also, "EXAMPLE" outside of the bloc?

File: january-slates/readme.md [hunk 3]
```diff
@@ -1 +1,106 @@
...
+* format of most of the information - visually how is the textual information presented in the slate? e.g. "boxes to fill in", "key-value pairs", "free text"  
+* (comma)  
+* Anything moving on screen during slate? - are there things like animations and other things that are moving during the slate information
+
+_Note: in some columns there is a comma within the raw value of the column._
+
+### Decisions and Differentiations during Annotation
+**What denotes start and end** - Decisions as to what exactly constitutes the start and end of the slate times is unknown. Is it when it first starts fading in, or when its fully done fading? 
+
+**Errors in the raw** - There are errors in the raw format, notably, in `CLAMS_slate_annotation_metadata.csv` line 203 "cpb-aacip-394-150gbd75", column(Writing Types F) is a typo "typeed" instead of "typed".
```
Comment (keighrim) [inline january-slates/readme.md:88]: Probably a good idea to create a hyperlink to that line of that file of that commit as a permalink. e.g. `https://github.com/clamsproject/aapb-annotations/blob/f884e10d0b9d4b1d68e294d83c6e838528d2c249/january-slates/230101-aapb-collaboration-7/CLAMS_slate_annotation_metadata.csv?plain=1#L203`

**Review by keighrim — CHANGES_REQUESTED** [review]

File: january-slates/readme.md [hunk 4]
```diff
@@ -1 +1,106 @@
...
+[More information about slates](https://www.youtube.com/watch?v=Heg6kDxXZ8k&t=13).  
+[What goes on a slate](theblackandblue.com/2012/11/05/deciphering-film-slate-1/).  
+This project creates a dataset that annotates where informational frames about the video are located timewise from many videos/GUIDs. 
+This dataset is needed to power automatic detection of slate information in the video collection at the AAPB. 
+Many of the collection pieces have incomplete or unverified information about what the video is. 
+This detected information could be used to verify and update metadata about that collection piece.
+
+### Project Information
+```
+Annotation project name - January Slates
```
[the remark this answers is not in the document]
Comment (jarumihooi) [reply january-slates/readme.md:20]: Ok, in that 'codebox' i will change all to reflect the exact title of the dir. 

Do you want to do anything differently or change any names?

File: january-slates/readme.md (hunk 1 above)
Comment (keighrim) [reply january-slates/readme.md]: I was thinking something like this, with bullet points and indentations; 

---
### Data Overview
* INPUT - a set of video files to be annotated. _TODO: Which tool?_   
* INTERMEDIATE RAW OUTPUT - `.csv` file where each line is the time of when the slate frames appear in that video.
* Format
    * Fields: GUID, ..., Anything moving on screen during slate?
    * Example:
        ```
        cpb-aacip-81-881jx33t,",","Woman                     ,","00:00:00;00 ,","00:00:05;04 ,","handwriting  ,",recorded,,,",",boxes to fill in,",",no,
        cpb-aacip-41-34fn32g7,",","Carolina Journal          ,","00:00:00;00 ,","00:00:14;28 ,","typed        ,",digital?,,,",",key-value pairs,",",countdown,
        ```
        > [!NOTE]
        > Note the extra commas added in and extra blank columns and extra space within the column names. 

* FINAL PREPARED GOLD OUTPUT - `.tsv` file that seems to be actually comma-separated anyway. The gold files conform to the repository readme guideline that each gold must relate to only one GUID. Therefore, each of these gold files is only 1 video/GUID each. Note: the column headers are added to each file.  
    * FORMAT 
        * Fields: GUID, ..., Anything moving on screen during slate?
        * EXAMPLE:
            ```
            GUID,Series/Group,Slate Start,Slate End,Writing Types,Recorded/Digital,format of most of the information,Anything moving on screen during slate?
            cpb-aacip-29-01pg4g2x,Prime Time Wisconsin,00:00:00;00,00:00:08;14,typed,digital,key-value pairs,no
            ```

Comment (jarumihooi) [reply january-slates/readme.md]: Ok, I've changed it to match.

**Comment by jarumihooi** [comment]
Should this non-comprehensive guide on slate types be downloaded into pdf and added to the dir? 
https://docs.google.com/document/d/1Xf43EpVzQbIOB-7KTadEyU3eam9xIvLlSGkjy4Ff2v4/edit

**Review by keighrim — COMMENTED** [review]
I updated some cosmetics, and re-did the data section with updated `process.py`. Let me know what you think. (also now that the file in this branch is diverged from the gist version, I think it might be a good idea to take the gist down to avoid any confusion)

File: january-slates/readme.md [hunk 5]
```diff
@@ -13,11 +13,11 @@ This dataset is needed to power automatic detection of slate information in the
 Many of the collection pieces have incomplete or unverified information about what the video is. 
 This detected information could be used to verify and update metadata about that collection piece.
 
-### Project Information
+### Specs
 * Annotation project name - `january-slates`
 * Annotator Demographics
     * Number of annotators - Two
-    * Occupational description - Two Public Broadcasting Organization Media Librarians and Volunteers
+    * Occupational description - Two Public Broadcasting Organization git Media Librarians and Volunteers
```
Comment (keighrim) [inline january-slates/readme.md:20]: typo?

**Review by keighrim — APPROVED** [review]

File: january-slates/readme.md (hunk 5 above)
Comment (keighrim) [reply january-slates/readme.md:20]: Why didn't this get fixed before merge?

**Comment by keighrim** [comment]
This PR was merged, then the `main` history was re-written after that, obsoleting the history in this PR. See #51 for overwriting PR.

File: january-slates/readme.md (hunk 5 above)
Comment (jarumihooi) [reply january-slates/readme.md:20]: This got confused as fixed already. Will have to fix on next commit.

Comment (keighrim) [reply january-slates/readme.md:20]: fixed in 34978bccf3cb9d6e3b0ccfe3c6c0cf15d2403f2d

**Pull request merged**